aten/src/ATen/native/sparse/SparseTensorMath.cpp

```diff
@@ -164,6 +164,7 @@ Tensor& div_sparse_(Tensor& self, const Tensor& value) {
 SparseTensor& div_out_sparse_zerodim(SparseTensor& r, const SparseTensor& t, const Tensor& value) {
   TORCH_CHECK(value.dim() == 0, "sparse division only supports division by a scalar (got shape ",
       value.sizes(), " for argument 'other')");
+  TORCH_CHECK(!value.is_sparse(), "Unsupported tensor layout: Can't divide a sparse tensor by a sparse value");
```

Comment:
nit: "sparse value" sounds like it is a scalar, not a tensor. Furthermore, users won't really know that the second argument to `div` is called `value`. Could we just say "can't divide a sparse tensor by a sparse tensor"?

Reply:
it is sort of supposed to be a "value" though ... you can't pass a dimensioned tensor of any layout (as asserted above this line)

Reply:
Ah, my bad, I missed that.

Reply:
updated again to be clearer anyway